packages/ui/src/components/molecules/PremiumFeature/PremiumFeature.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactNode } from 'react';
+import { Crown } from 'lucide-react';
+
+import { Card, CardContent, CardFooter, CardHeader, Image } from '@/components';
+
+export const PremiumFeature = ({ footer, content }: { footer: ReactNode; content: ReactNode }) => (
+  <Card
+    className={`bg-background absolute right-6 top-5 flex max-h-[328px] max-w-[293px] flex-col rounded-lg border-[1px] border-[#E3E0E9] p-6 2xl:right-[4.5rem]`}
```

Comment:
If this component stands on its own it might be a better idea to pass the absolute and top/right as props.